services/galley/test/integration/API/Teams.hs

```diff
@@ -156,6 +159,40 @@ testCreateTeamWithMembers = do
       e ^. eventTeam @?= (team ^. teamId)
       e ^. eventData @?= Just (EdTeamCreate team)
 
+testListTeamMembersDefaultLimit :: TestM ()
+testListTeamMembersDefaultLimit = do
+  (owner, tid) <- createTeam
```

Comment:
perhaps make that name explicit, too?

```suggestion
  (owner, tid) <- createBindingTeam
```

Reply:
I am afraid it will break more things, I will make this change separately.